elife-00666.xml

```diff
@@ -1797,7 +1797,7 @@ not add all rids to the xref as it does not work. Only add the first one -->
                                     <title>Title of the source code.</title>
                                     <p>Legend of the source code.</p>
                                 </caption>
-                                <media mimetype="application" mime-subtype="xml" xlink:href="elife-00666-fig4-code1.xlsx"/>
+                                <media mimetype="application" mime-subtype="xlsx" xlink:href="elife-00666-fig4-code1.xlsx"/>
```

Comment:
Ok, I'll just leave this as a warning that it might cause problems for anyone trying to use it.

@gnott, I'll open a ticket as we'll need to turn these into valid values when parsing.